### PR TITLE
Renaming Oracle check (DBMON-3594)

### PR DESCRIPTION
**Pull request by nenadnoveljic**
### What does this PR do?

Renaming `oracle-dbm` check to `oracle`

### Motivation

Deprecating the current Python `oracle` check. The functionality of the Go check is a superset of the Python check, and it's easier to maintain.

### Additional Notes

The Oracle Go check requires more privileges than the Python check. If the privileges are missing, the check will detect it and emulate the functionality of the legacy Oracle check.

### Possible Drawbacks / Trade-offs

The goal is to make the change as non-disruptive. In case of errors for there are no workarounds, we introduced the parameter `loader` which can be set to `python`, which will load the old integration. We will not document this parameter. The support would instruct the customer to set it if needed.

### Describe how to test/QA your changes

1. The configurations are loaded from both the `oracle-dbm.d` and `oracle.d` directories.
2. For configurations in `oracle.d` test if Python check is loaded when `loader=python` and `dbm=false`

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -103,4 +103,4 @@
     "dca-1.9.0": {
         "SECURITY_AGENT_POLICIES_VERSION": "v0.3"
     }
-}
```
Comment (paulcacheux) [inline release.json:106]: Don't think we really need this change

**Comment by vickenty** [comment]
Please also update paths in .github/CODEOWNERS

File: cmd/agent/dist/conf.d/oracle.d/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+The configuration directory ``oracle-dbm.d`` is deprecated. Use ``oracle.d`` instead.
```
Comment (vickenty) [inline cmd/agent/dist/conf.d/oracle.d/README.md:2]: Would it make sense to place this `oracle-dbm.d/README.md` too?

File: tasks/build_tags.py [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "netcgo",  # Force the use of the CGO resolver. This will also have the effect of making the binary non-static
     "npm",
     "oracle",
+    "oracle-dbm",
```
Comment (vickenty) [inline tasks/build_tags.py:37]: This build tag is not mentioned anywhere. Is it intentional?

File: pkg/commonchecks/corechecks.go [hunk 4]
```diff
@@ -78,6 +79,7 @@ func RegisterChecks(store workloadmeta.Component) {
 	corecheckLoader.RegisterCheck(network.CheckName, network.Factory())
 	corecheckLoader.RegisterCheck(nvidia.CheckName, nvidia.Factory())
 	corecheckLoader.RegisterCheck(oracle.CheckName, oracle.Factory())
+	corecheckLoader.RegisterCheck(oracledbm.CheckName, oracle.Factory())
```
Comment (vickenty) [inline pkg/commonchecks/corechecks.go:82]: Would it be possible to place old check name in the new package? I don't think we need a separate package just for a single constant.

**Comment by nenadnoveljic** [comment]
> Please also update paths in .github/CODEOWNERS

Done, thanks!

File: cmd/agent/dist/conf.d/oracle.d/README.md (hunk 2 above)
Comment (nenadnoveljic) [reply cmd/agent/dist/conf.d/oracle.d/README.md:2]: Done!

**Review by jhgilbert — APPROVED** [review]

File: tasks/build_tags.py (hunk 3 above)
Comment (nenadnoveljic) [reply tasks/build_tags.py]: I removed them.

File: pkg/commonchecks/corechecks.go (hunk 4 above)
Comment (nenadnoveljic) [reply pkg/commonchecks/corechecks.go]: Right, thanks. I changed it.

**Review by lu-zhengda — APPROVED** [review]

**Review by louis-cqrl — APPROVED** [review]
LGTM for files ASC owns

**Review by vickenty — APPROVED** [review]

**Pull request merged**